doc/submitting.md

```diff
@@ -0,0 +1,27 @@
+## Submitting changes to SUSE Build Service
+
+Submissions are managed by a SUSE internal jenkins node in the InstallTools tab.
```

Comment:
I see... But I don't think it is a problem, the machine is not available from outside anyway.